tests/console/upgrade_snapshots.pm

```diff
@@ -23,12 +23,17 @@ sub run {
     select_console 'root-console';
 
     my $waittime = 200 * get_var('TIMEOUT_SCALE', 1);
-    script_run("snapper list | tee /dev/$serialdev", 0);
+
+    # 'snapper list' will cost a lots of time to finish if btrfs-balance.sh was triggered
+    # To make it faster, we'd check if snapper have disable-used-space option.
+    # This is a workaround for bsc#1167353.
+    record_info('workaround for bsc#1167353');
```

Comment:
`btrfs balance status /` would show you if balancing is going on, I am sure. Ok, leave it as is.

Reply:
thanks a lot